config/azure/options.windows

```diff
@@ -45,7 +45,6 @@ GLEW_INCLUDE_PATH = deps + "\\include"
 GLEW_LIB_SUFFIX = "32"
 
 TBB_INCLUDE_PATH = deps + "\\include"
-USD_LIB_PREFIX = "disable_usd"
```

Comment:
Are you sure this is doing what you think? I don't see a Windows variant in the Azure checks of this PR, and the tests still aren't running for the GitHub Actions on Windows.

Reply:
Ah, nevermind, I see I'm mistaken and the azure options files are still used in GitHub CI https://github.com/ImageEngine/cortex/blob/master/.github/workflows/main.yml#L19-L22

Reply:
> Are you sure this is doing what you think? I don't see a Windows variant in the Azure checks of this PR, and the tests still aren't running for the GitHub Actions on Windows.

I think so, it looks to be building IECoreUSD in the Github Actions CI:
https://github.com/ImageEngine/cortex/pull/1113/checks?check_run_id=1651827885#step:10:57

Right now the build options for Github Actions are also coming from the `config/azure/options.*` 

The original motivation for `USD_LIB_PREFIX="disable_usd"` was a kludge to get Cortex passing the CI in #1105 while I knew that USD was not working. This enables it again to check for the build succeeding.

Reply:
Ah, beat me to it!